### DISCUSSION TITLE
发送语音问题

### DISCUSSION THREAD

**Discussion by gogolrq** [GraiaProject/Application#133] [Q&A]
尝试使用voice fromlocalfile， 发现发送了一个空语音。求助解决办法，多谢。

**Comment by raines1220** [comment]
同问，我也在尝试发语音。但是发的是一个空语音。

**Comment by I-love-study** [comment]
现在的mirai仅仅支持silk和amr的音频发送
amr的话可以使用ffmpeg等库进行压制，但是音质效果不如silk
silk的音质至少相比于amr好一些，不过能压制音频的库有点少
不建议的话可以试试鄙人写的`graiax-silkcoder`库，可以转
(好像mirai一直有silk发送有问题的issue,所以不要指望音质有多好)

**Reply by gogolrq** [reply]
多谢大佬！

**Comment by Mechorca** [comment]
请问楼主解决了吗。

**Reply by gogolrq** [reply]
只能发amr
